src/components/Create/components/RecallCard/hooks/FundingCycle.ts

```diff
@@ -0,0 +1,44 @@
+import { useAppSelector } from 'hooks/AppSelector'
+import { deriveDurationUnit, secondsToOtherUnit } from 'utils/format/formatTime'
+
+const formatUnit = ({ unit, plural }: { unit: string; plural: boolean }) => {
+  let formatted = unit.charAt(0).toUpperCase() + unit.slice(1)
+  if (formatted.endsWith('s')) {
+    if (!plural) {
+      formatted = formatted.slice(0, -1)
+    }
+  }
+  return formatted
+}
+
+const formatFundingCycle = (duration: string) => {
+  const durationAsNumber = parseInt(duration)
+  if (isNaN(durationAsNumber)) return 'No duration'
+
+  const derivedUnit = deriveDurationUnit(durationAsNumber)
+  const formattedDuration = secondsToOtherUnit({
+    duration: durationAsNumber,
+    unit: derivedUnit,
+  })
+
+  const formattedUnit = formatUnit({
+    unit: derivedUnit,
+    plural: formattedDuration > 1,
+  })
+  return `${secondsToOtherUnit({
+    duration: durationAsNumber,
+    unit: derivedUnit,
+  })} ${formattedUnit}`
+}
+
+export const useFundingCycle = () => {
```

Comment:
**suggestion** wdyt about something like `useFundingCycleRecallValue` or similar

Reply:
love it - will make the change